README.md

```diff
@@ -220,6 +212,5 @@ I would like to thank Microverse for providing me with the reading material that
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on this milestone and updating the ReadMe as requested by previous reviewer. Unfortnately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/62849872/218951213-3426ac8e-2418-42a6-9e77-67fb6c1abf9c.png)